plugin/storage/cassandra/dependencystore/storage.go

```diff
@@ -93,6 +98,14 @@ func (s *DependencyStore) GetDependencies(endTs time.Time, lookback time.Duratio
 	return mDependency, nil
 }
 
+func getDepSelectString(startTs time.Time, endTs time.Time) string {
```

Comment:
Cassandra has no date range support?

Reply:
Cassandra does not support range queries on partition keys

Reply:
OK that explains it.